### PR TITLE
Adjusts reports to new groups concept for benchmarks

### PR DESCRIPTION
**Pull request by tanertopal**

### CODE REVIEW

File: xain/benchmark/exec/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# benchmark execution
+
+# Ploting
```
Comment (danieljanes) [inline xain/benchmark/exec/README.md]: Plotting

File: xain/benchmark/exec/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# benchmark execution
```
Comment (danieljanes) [inline xain/benchmark/exec/README.md]: Benchmark Execution

File: xain/benchmark/exec/report.py [hunk 3]
```diff
@@ -21,21 +22,32 @@
 FLAGS = flags.FLAGS
 
 
-def read_accuracies_from_results(dname: str):
+def get_task_class(data: dict) -> str:
```
[the remark this answers is not in the document]
Comment (tanertopal) [reply xain/benchmark/exec/report.py]: done

File: xain/benchmark/exec/report.py [hunk 4]
```diff
@@ -21,21 +22,32 @@
...
+    return data["task_name"].split("_")[0]
+
+
+def get_task_label(data: dict) -> str:
```
[the remark this answers is not in the document]
Comment (tanertopal) [reply xain/benchmark/exec/report.py]: done

**Pull request merged**